### ISSUE TITLE
HTTP/HTTPS urls doesn't work

### ISSUE THREAD

**Issue by deamme** [feross/webtorrent#387]
States in the documentation that `torrentId` can be one of the following:
- magnet uri (utf8 string) - 
- torrent file (buffer)
- info hash (hex string or buffer)
- parsed torrent (from parse-torrent)
- http/https url to a torrent file (string) - Doesn't work
- filesystem path to a torrent file (string)

**Comment by feross** [comment]
Are you trying this in the browser? Right now http/https urls to .torrent files don't work there because I completely excluded the `simple-get` module from the browser. This module makes make the http request, and includes `http`, which is pretty huge in terms of file size.

Most domains will probably not have the CORS headers configured correctly to allow your website to download the .torrent file. Do you think it's worth adding 50K gzipped, almost doubling the size of the webtorrent library for this feature?

**Comment by deamme** [comment]
@feross I see the problem and of course we shouldn't double the size of the library. I'll close this.

**Comment by feross** [comment]
Actually, maybe this wasn't entirely fair. `http` includes `stream` and `buffer` which are also huge, but those are already included in webtorrent.

```bash
$ browserify -r buffer | gzip | wc -c
   10836
$ browserify -r stream | gzip | wc -c
   31852
```

**Comment by deamme** [comment]
It could work out then but I don't really see any sites supporting CORS.

**Comment by feross** [comment]
Actually, I just checked and since we started supporting web seeding, the `http` module is already being pulled in, so this actually adds basically no file size.

We'll support it.

**Comment by feross** [comment]
Released as 0.54.0.
